src/wallet3/wallet.cpp

```diff
@@ -71,6 +71,7 @@ namespace wallet
   void
   Wallet::init()
   {
+    keys->expand_subaddresses({50, 200});
```

Comment:
Should definitely be configurable, but mercifully the scanning time doesn't really scale with this.